Conversion warning in TinyMT

With  -Wconversion I get a warning:
```
../thirdparty/alpaka-0.4.1/include/alpaka/rand/TinyMT/tinymt32.h:216:33: warning: conversion to ‘float’ from ‘uint32_t {aka unsigned int}’ may alter its value [-Wconversion]
     return (tinymt32_temper(random) >> 8) * TINYMT32_MUL;
        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
```
I fixed it by explicitly stating the conversion:
`
return float(tinymt32_temper(random) >> 8) * TINYMT32_MUL;
`

--- a/include/alpaka/rand/TinyMT/tinymt32.h
+++ b/include/alpaka/rand/TinyMT/tinymt32.h
@@ -213,7 +213,7 @@ inline static uint32_t tinymt32_generate_uint32(tinymt32_t * random) {
  */
 inline static float tinymt32_generate_float(tinymt32_t * random) {
     tinymt32_next_state(random);
-    return (tinymt32_temper(random) >> 8) * TINYMT32_MUL;
+    return float(tinymt32_temper(random) >> 8) * TINYMT32_MUL;
 }
 
 /**

